README.md

```diff
@@ -32,7 +34,143 @@ For good first issues regarding contributing, see [issues labeled with open sour
+
+### Frontend embed
+
+Our embed consists in [the survey form embedded in Helsingin Sanomat](https://www.hs.fi/kotimaa/art-2000006455647.html). Related files can be found in:
+
+```
+index-frontend-embed-v1tsx
+src/
+- frontend/
+-- embed/
+--- v1/
+```
+
+The embed has been developed using [jQuery 3.4.1](https://jquery.com/). The embed is also used in the frontend main app.
+
+### Common files
+
+There are files that are shared across the apps, e.g. `fonts.d.ts` and `src/common/`.
+
+## Prerequisites
+
+Project requires Node.js >=12 <13 to be installed; all other dependencies come from NPM via package.json.
+
+You will need Node and npm (npm comes with Node). We recommend [using a node version manager, nvm](https://github.com/nvm-sh/nvm) to install Node, as it allows automatically switching between node versions across projects.
+
+We recommend also installing [direnv](https://direnv.net/), which helps automatically set environment variables as you switch directories.
+
+## Development environment setup
+
+Clone the project from GitHub (make sure to clone it from the directory you wish to contain your project)
+
+```shell
+git clone git@github.com:futurice/symptomradar.git
+```
+
+Install NPM dependencies
+
+```shell
+cd symptomradar
+rm -rf node_modules
+nvm use
+npm i
+```
+
+(`rm -rf node_modules` is a completely optional step needed only if you have previously attempted to install the project and things went wrong)
+
+If you have some issues with the packages installation, please try the following:
+
+```shell
+rm -rf node_modules
+nvm use
+npm i --no-optional
+```
+
+## Environment variables
+
+This application uses a number of environment variables to customise the runtime behaviour. The following steps assume that you are using [direnv](https://direnv.net/).
+
+Copy over .envrc.sample into .envrc, at the project root
+
+```shell
+cp .envrc.sample .envrc
+```
+
+To work with the frontend, you shouldn't need any further variable than the ones already provided.
+Follow the instructions inside `.envrc` if you need to set up a mock API endpoint.
+
+Finally, you need to read the variables from `.envrc`:
+
+```shell
+source .envrc
+```
+
+This command will also make sure that you are using the right Node version for this project.
+
+## Available NPM tasks (frontend)
+
+- `frontend-main-start` runs the React application of Oiretutka
+- `frontend-main-build` creates a build for the React application
+- `frontend-embed-v1-start` runs the jQuery embed of Oiretutka
+- `frontend-embed-v1-build` creates a build for the jQuery embed
+
+## Wrap up
+
+For the following times you will work on the project, you will need to
+
+- enter the project folder
+- run `source .env`, which will automatically set up the right node version (assuming you are using nvm)
+- `npm run frontend-main-start` or `npm run frontend-embed-v1-start` according to which part of the project you want to run locally
+
+## Developing the stand alone (React) app
+
```

Comment:
@Doniee could you write a few words or lines about developing the React app here? 😄

Reply:
Heyooo! Can you check also my latest changes @sjarva ? The way I see it now, it feels a bit duplicated to have this here and then also those sections above to explain the project structure  (lines 47 to 84) -- I don't mind their placement, but I guess the information can be easily grouped together!

Reply:
Actually, nevermind, I see that you spent more time writing about all sorts of useful information! Disregard what I said! 😂 

And damn beautiful work!